rust/origen/src/core/status.rs

```diff
@@ -57,3 +60,12 @@ fn search_for_app_root() -> (bool, PathBuf) {
         (true, path)
     }
 }
+
+fn get_home_dir() -> PathBuf {
+    if cfg!(windows) {
+        PathBuf::from(env::var("USERPROFILE").expect("No environment variable for USERPROFILE"))
+    }
+    else {
+        PathBuf::from(env::var("HOME").expect("No environment variable for HOME"))
```

Comment:
I think we should phrase these as "Please set environment variable HOME to point to your home directory, then try again"